subgraph/src/helpers.ts

```diff
@@ -75,3 +75,16 @@ export function loadOrCreateUnlockDailyData(
 
   return unlockDailyData
 }
+
+export function getKeyManagerOf(
+  lockAddress: Address,
+  tokenId: BigInt,
+  owner: Address
+): Address {
+  const lockContract = PublicLockV7.bind(lockAddress)
```

Comment:
I am a bit confused about why you use `PublicLockV7` here? But i think that could work? @clemsos wdyt?

Reply:
there is no difference in the implementation of `keyManagerOf` from `v7` and `v11`, will works fine